### PR TITLE
Fixed Systers open source mailing list link text

### LINKED ISSUE #167: Change the text for Systers Open Source mailing list in Contact Us page
## Description
As a person interested in Systers' Open Source Mailing list,
I need see that the mailing-list is specific to Open Source related topics,
so that I can understand the scope of the emails I'm getting from the Systers Open Source mailing list.

On the [Contact Us](http://systers.io/contactus.html) page, the link for the [Systers' Open Source mailing list](http://systers.org/mailman/listinfo/systers-opensource) has the text "Systers Mailing List". This can be misleading since there is also the [Systers mailing list](http://systers.org/mailman/listinfo/systers), which is not only related to the Open Source community.

In the [Join Systers](http://systers.io/index.html#join-systers) section on the main page it says "systers-dev mailing list", which is also an alternative.

## Mocks

Actual screen:

image

Expected screen:

image

### Update
- [ ] [/contactus.html](https://github.com/systers/systers.github.io/blob/master/contactus.html) fix text for the mailing list link

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
5 minutes

### PR DESCRIPTION
**Pull request by isabelcosta**
### Description
I just changed the text for the Systers Open Source mailing list and the Systers Open Source Slack link, on the [Contact Us](http://systers.io/contactus.html) page.

Fixes #167 

### Type of Change:

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

I ran the website locally as explained on README. 
Here's the updated screen:

image

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: contactus.html [hunk 1]
```diff
@@ -77,7 +77,7 @@
         <li class="horizontal"><a href="http://systers.io/slack-systers-opensource/" target="_blank">Slack</a></li>
         <li class="horizontal"><a href="https://github.com/systers" target="_blank">GitHub</a></li>
         <li class="horizontal"><a href="https://twitter.com/systers_org" target="_blank">Twitter</a></li>
-        <li class="horizontal"><a href="http://systers.org/mailman/listinfo/systers-opensource" target="_blank">Systers Mailing List</a></li>
+        <li class="horizontal"><a href="http://systers.org/mailman/listinfo/systers-opensource" target="_blank">Systers Open Source mailing list</a></li>
```
Comment (sara-02) [inline contactus.html]: @isabelcosta let us be consistent with the capitalization and use capital `Mailing List` instead.

Comment (isabelcosta) [reply contactus.html]: Of course @sara-02 , I'll change that now

**Comment by sara-02** [comment]
@isabelcosta Looks good now, please squash the commits into a single commit.

**Comment by isabelcosta** [comment]
@sara-02 I'm now trying to squash the commit locally

**Comment by sara-02** [comment]
@divyanshu-rawat we can merge this now.

**Comment by divyanshu-rawat** [comment]
okay, @sara-02 I am merging this 👍

**Pull request merged**